### DISCUSSION TITLE
Testing offline in local dev mode

### DISCUSSION THREAD

**Discussion by Yuripetusko** [DuCanhGH/next-pwa#165] [Q&A]
Currently the plugin uses `ctx.webpackContext.dev` to determine if resources should be cached or not, so there's no way to test offline while running `next dev`, wouldn't it be better to just have extra config option "disableInDev?: boolean" which would be `ctx.webpackContext.dev` by default if `disableInDev` is undefined, but can be overwritten by dev if they really want to test it? Or this would create additional issues and complexities of caching webpack generated chunks?

**Comment by simonorzel26** [comment]
Seems reasonable, whats the reason for not having this?
